insights_analytics_collector/collection.py

```diff
@@ -67,7 +68,10 @@ def gather(self, max_data_size):
 
             self.gathering_successful = True
         except Exception as e:
-            self.logger.exception(f"Could not generate metric {self.filename}: {e}")
+            self.logger.exception(
```

Comment:
what's the reason for replacing simple string with multi-line strings with `#nopep8` exceptions?